tools/tune/eq/gui.m

```diff
@@ -1,13 +1,104 @@
-clear;
+function gui()
+  pkg load signal;
+  clear;
+  EQ = [];
+  Fs = get_config().Fs;
+  nyquist_f = Fs/2;
+  fn = 1;
+  % create figure and panel on it
+  fn = figure(fn, "toolbar", "none", "menubar", "none", 'Position', [200 200 1500 700]); fn = fn + 1;
+  EQ_right = uipanel ("title", "EQ Right", "position", [0, 0, 1, 0.5]);
+  EQ_left = uipanel ("title", "EQ Left", "position", [0, 0.5, 1, 0.5]);
+  main_panes = [struct("pane", EQ_left); struct("pane", EQ_right)];
+
+  % UI panel relative position table
+  positions = [
+    [0    0   0.25 0.5 ]
+    [0.25 0   0.5  0.5 ]
+    [0.5  0   0.75 0.5 ]
+    [0.75 0   1    0.5 ]
+    [0    0.5 0.25 1   ]
+    [0.25 0.5 0.5  1   ]
+    [0.5  0.5 0.75 1   ]
+    [0.75 0.5 1    1   ]
+  ];
+
+  fn = figure(fn); fn = fn + 1;
+  h = [0 0];
+  w = [1 nyquist_f];
+  n_octaves = 10;
+  subplot(2, 1, 1);
+  main_panes(1).plot = plot(log2(w), h, "linewidth", 4);
+  % Range set to match http://audio-tuning.appspot.com/
+  axis([log2(nyquist_f / (2 ^ n_octaves)), log2(nyquist_f), -24, 18]);
```

Comment:
Can you add after this also "grid on;" for nicer looking plot.

Reply:
Also could we have plot scale in logarithmic Hertz instead of octave numbers? Or support both via config struct option. That's also something I can contribute later so not critical.

Reply:
I will add a TODO for the axis. Initially I want to make sure it follows our tools then extend from there.

Reply:
Yep, that sounds like a good plan.